service/repository/csv.go

```diff
@@ -0,0 +1,108 @@
+package repository
+
+import (
+	"bufio"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/etyberick/golang-bootcamp-2020/entity"
+)
+
+type quoteRepository struct {
+	file   os.File
+	quotes []entity.Quote
+}
+
+// EmptyDatabase message response
+const EmptyDatabase = "database is empty"
```

Comment:
```suggestion
const ErrEmptyDatabase = "database is empty"
```

Just to make it crystal clear that this is an error message.

Reply:
Done